packages/create-catalyst/src/utils/cli-api.ts

```diff
@@ -28,7 +28,12 @@ export class CliApi {
     });
   }
 
-  async createChannel(name: string, installSampleData = false) {
+  async createChannel(
+    name: string,
+    storefrontLanguage: string,
+    additionalLocales: string[],
```

Comment:
Can we align on either "language" or "locale" here? e.g. could it be `storefrontLocale` instead?

Reply:
I will rename `storefrontLanguage` to `storefrontLocale`